### PR TITLE
metrics: Update machine learning documentation

### LINKED ISSUE #7359: metrics: update machine learning documentation

### PR DESCRIPTION
**Pull request by GabyCT**
This PR updates the machine learning documentation related with Tensorflow and Pytorch benchmarks.

Fixes #7359

### CODE REVIEW

**Review by fidencio — APPROVED** [review]
lgtm, thanks @GabyCT!

File: tests/metrics/machine_learning/README.md [hunk 1]
```diff
@@ -3,7 +3,10 @@
 Kata Containers provides a series of performance tests using the
 TensorFlow reference benchmarks (tf_cnn_benchmarks).
 The tf_cnn_benchmarks containers TensorFlow implementations of several
-popular convolutional models.
+popular convolutional models https://github.com/tensorflow/benchmarks/tree/master/scripts/tf_cnn_benchmarks.
+
+Currently the Kata Containers TensorFlow benchmark includes test for
```
Comment (dborquez) [inline tests/metrics/machine_learning/README.md:8]:
```suggestion
Currently the TensorFlow benchmark on Kata Containers includes
```

**Review by dborquez — CHANGES_REQUESTED** [review]

File: tests/metrics/machine_learning/README.md (hunk 1 above)
Comment (dborquez) [inline tests/metrics/machine_learning/README.md:8]:
```suggestion
Currently the TensorFlow benchmark on Kata Containers includes
```

**Review by dborquez — APPROVED** [review]
I've included a suggestion, but see if it looks ok for you.

**Comment by GabyCT** [comment]
@dborquez changes applied

**Pull request merged**